packages/wrangler/src/config/config-helpers.ts

```diff
@@ -128,7 +128,7 @@ function findRedirectedWranglerConfig(
 			}
 		}
 
-		logger.warn(dedent`
+		logger.info(dedent`
```

Comment:
Purely aesthetic, but it feels like it might be nice to indent a bit to make the grouping clearer? Something like:

```
Using redirected Wrangler configuration:
  Configuration being used: ...
  Original user's configuration: ...
  Deploy configuration file: ...
```

Reply:
OK

Reply:
done